background.ts

```diff
@@ -124,6 +126,8 @@ mainEvents.on('settings-update', async(newSettings) => {
     pathManager = getPathManagerFor(newSettings.pathManagementStrategy);
     await pathManager.enforce();
   }
+
+  await runRdctlSetup(newSettings);
```

Comment:
This should only be called on windows, right?

Reply:
Once #3808 and #3842 are merged, this will need to be called on all three platforms. At the moment (and this was an oversight), calling `rdctl` as it is here will produce a nonzero return code on Linux and macOS. This is because `EnsureAutostart` returns a "not implemented" error on these platforms. I'm going to change this so that these function calls are no-ops, which will allow `runRdctlSetup` to run without errors.